nirc_ehr/resources/referenceStudy/study/datasets/datasets_metadata.xml

```diff
@@ -58,6 +58,14 @@
             <column columnName="species">
                 <datatype>varchar</datatype>
             </column>
+            <column columnName="lab">
```

Comment:
The name "lab" could be confusing if you're combining this data in reports with data from other datasets. Other centers have used "origin". Does that seem like it would be a better fit for your use case?

Reply:
origin for us is more like country or geographic area. I can be a bit more direct and call it sourceFacility